create_app.ts

```diff
@@ -0,0 +1,179 @@
+import { green, red } from "./deps.ts";
+const { args } = Deno
+const wantsHelp = (args.find(arg => arg === '--help') !== undefined)
+const wantsWebApp = (args.find(arg => arg === '--web-app') !== undefined)
+const wantsApi = (args.find(arg => arg === '--api') !== undefined)
+const wantsVue = (args.find(arg => arg === '--with-vue') !== undefined)
+const cwd = Deno.cwd()
+// strip this file name from the path and add the link to the boilerplate dir
+const boilerPlateDir = (import.meta.url.slice(0, -13)).substring(5) + 'console/create_app'
+//                                  ^^^^^^^^              ^^^^            ^^^^^^^
+//                Remove the file name from the path   Strip "file://"  Add boiler plate dir
+const notesForUser: string[] = []
+
+//////////////////////////////////////////////////////////////////////////////
+// FILE MARKER - FUNCTIONS ///////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/**
+ * Responsible for showing the help message when a user uses `--help`
+ */
+function showHelp () {
+    const helpMessage =
+        '\n' +
+        'A create app script for Drash' +
+        '\n' +
+        '\n' +
+        'USAGE:' +
+        '\n' +
+        '    deno run --allow-read --allow-run create_app.ts [OPTIONS]' +
+        '\n' +
+        '    deno run --allow-read --allow-run https://deno.land/x/drash/create_app.ts [OPTIONS]' +
+        '\n' +
+        '\n' +
+        'OPTIONS:' +
+        '\n' +
+        'The --api and --web-app options cannot be used together' +
+        '\n' +
+        '\n' +
+        '    --api' +
+        '\n' +
+        '        Creates the file structure and content for a Drash API' +
+        '\n' +
+        '\n' +
+        '    --web-app, --web-app --with-vue' +
+        '\n' +
+        '        Creates the file structure and content for a Web App using Drash.' +
+        '\n' +
+        '        Will add a Vue foundation if the --with-vue option is passed in' +
+        '\n' +
+        '\n' +
+        'EXAMPLE:' +
+        '\n' +
+        '    mkdir my-drash-api' +
+        '\n' +
+        '    cd my-drash-api' +
+        '\n' +
+        '    deno run --allow-read --allow-run https://deno.land/x/drash/create_app.ts --api' +
+        '\n'
+    Deno.run({
+        cmd: ['echo', helpMessage]
+    })
+}
+
+/**
+ * On writing to and creating files, we send a message to stdout to let the user know something
+ * is happening
+ *
+ * @param {string} message Message to show in the console. Required.
+ */
+function writeFileWrittenOrCreatedMessage (message: string) {
+    Deno.run({
+        cmd: ['echo', green(message)]
+    })
+}
+
+/**
+ * Send our thank you message for using it
+ */
+function sendThankYouMessage () {
+    notesForUser.push('Run your application: deno run --allow-run --allow-net --allow-read app.ts')
```

Comment:
do we need --allow-run here?

Reply:
No, not sure why that is in there then but have removed it: https://github.com/drashland/deno-drash/pull/217/commits/4d04ccfc3c9a8247eb5a7f3d43717a76fa9693f1